src/api/v1/authorization/routes/logout.ts

```diff
@@ -0,0 +1,20 @@
+import { FastifyReply, FastifyRequest } from 'fastify'
+import { getSession, terminateSession } from '../../../../security/session'
+
+export default async (req: FastifyRequest < { Body: {} }>, res: FastifyReply): Promise<void> => {
+  if (!req.headers.authorization) {
+    await res.status(500).send({ message: 'Error during logout' })
+    logger.error('Authorization header not present')
```

Comment:
This part is both unnecessary and supplies a misleading error code. The logout route already uses the requiresAuth middleware, meaning this check will never return true (as the middleware will terminate the request if the header doesn't exist).